pitch.md

```diff
@@ -46,24 +92,34 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There aren't any other DSLs in this domain that I know of. Automated coffee machines already have a GUI that lets you customize your drink in this way but
+
+1. It almost certainly doesn't compile to a DSL (I guess **maybe** it does, but I highly doubt it).
+2. The "language" has limited expressiveness. You can choose a base drink, and then modify the amount of coffee, sugar, and milk to add, but that's about it. The language I'm proposing would allow you to create a drink from the bottom up. 
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
-
 ### Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think I'll be spending a lot of time making language design decisions, but also a fair amount of time implementing the language. The free free-form nature of the language (in particular, the fact that you could probably make programs of arbitrary length) means that parsing will be something of a challenge - I'll probably have to get really familiar with parser combinators. Luckily, once I have the intermediate representation of a program, execution should be pretty easy - just make the relevant library calls with the parameters extracted from the program. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I'd call this a medium sized idea. Implementing it is going to be difficult (because there will be a lot of language design choices to make), but I think it'll be manageable over the course of the semester. That said, I tend to underestimate the difficulty of things, and debugging can end up a fair amount of one's time, which could turn this medium-sized idea to a pretty big one. Also, I learned in picobot-external that there is usually a **lot** of error handling that you can add to a lanugage, so if I happen to get lucky and finish way ahead of time, I could always sink a bunch of time into that.
 
 ### Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
 
+This is a good project because I'm really excited about it, and it could be a really useful idea! Also, there are a lot of language design decisions to make (though I feel like I've made some of the major ones already, I'm sure some of those decisions will change as they come in contact with the real world), making it a project well suited for this class. 
```

Comment:
I think the project is really cool but I want to see how it can be made more complex (maybe different user preferences the machine could have). I feel like it needs to extend past just a list of instructions to fill the entire rest of the semester.